roles/servicetelemetry/tasks/_local_signing_authority.yml

```diff
@@ -8,7 +8,6 @@
         name: '{{ ansible_operator_meta.namespace }}-selfsigned'
         namespace: '{{ ansible_operator_meta.namespace }}'
       spec:
-        duration: '{{ certificate_duration }}'
```

Comment:
With this removal does this mean that the self-signed issuer will need to regenerate every 90(?) days? The certificate_duration was recently added I believe, so I'm not sure the implications of removing it here on the overall schema.

Reply:
Victoria asked me about this out-of-band and we agree that my addition of the duration property to the Issuer object was a mistake. That property does not appear to exist[1], and we expect this change to have no effect.

That said, since we have no unit-test coverage, I would appreciate seeing the certificates that are generated by an altered duration in order to prove that this change (and really the whole PR) is behaving as expected.

[1] https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.IssuerConfig

Reply:
This idea feels like it might be better suited to https://github.com/infrawatch/functional-tests/ where we do a series of `oc patch` commands to exercise all the documented features, rather than the smoketest which is more focused on ensuring the default config stands up and appears to basically operate end to end.

Reply:
+1 I was performing manual testing and also looking to see if there is a good suite I can add testing for this. I'll try adding some coverage in the functional tests suite in my follow up patch.